### DISCUSSION TITLE
Welcome to ESP32MX1508 Discussions!

### DISCUSSION THREAD

**Discussion by ElectroMagus** [ElectroMagus/ESP32MX1508#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  Please let me know of any issues, comments or feedback regarding this library.

**Comment by pbecker83** [comment]
Great library! I tried setting this up for two motors to no avail. Any info would be appreciated.

Patrick 

"#include <ESP32MX1508.h>

#define PINA 9
#define PINB 10
#define PINC 2
#define PIND 3
#define CH1 0                   // 16 Channels (0-15) are availible
#define CH2 1                   // Make sure each pin is a different channel and not in use by other PWM devices (servos, LED's, etc)
#define CH3 2                   // 16 Channels (0-15) are availible
#define CH4 3
   
// Optional Parameters
#define RES 8                   // Resolution in bits:  8 (0-255),  12 (0-4095), or 16 (0-65535)     
#define FREQ  5000              // PWM Frequency in Hz    

MX1508 motorA(PINA,PINB, CH1, CH2);                       // Default-  8 bit resoluion at 2500 Hz
MX1508 motorB(PINC,PIND, CH3, CH4);                       // Default-  8 bit resoluion at 2500 Hz

//MX1508 motorA(PINA,PINB, CH1, CH2, RES);                // Specify resolution
//MX1508 motorA(PINA,PINB, CH1, CH2, RES, FREQ);          // Specify resolution and frequency

int FSR = 0;

void setup() {}

void loop() {
"

**Reply by ElectroMagus** [reply]
Thanks for posting the question! Can you post a copy of the code you are using and also a picture of your wiring? That might help me to understand what's going on.
